mesh/mesh.go

```diff
@@ -641,7 +594,7 @@ func (msh *Mesh) getAggregatedLayerHash(layerID types.LayerID) (types.Hash32, er
 	return hash, fmt.Errorf("get from DB: %w", err)
 }
 
-func (msh *Mesh) extractUniqueOrderedTransactions(l *types.Layer) (validBlockTxs []*types.Transaction) {
+func extractUniqueOrderedTransactions(l *types.Layer, mdb *DB, logger log.Log) (validBlockTxs []*types.Transaction) {
```

Comment:
why do you prefer to pass logger as a last value? i think we need some consistent approach

i am usually using it as a first argument as to me the idea is the same as with ctx. so logger logically can be a part of context

Reply:
i don't actually have a preference. changed.